index.js

```diff
@@ -0,0 +1,81 @@
+const Asana = require("asana");
+const passport = require("passport");
+const AsanaStrategy = require("passport-asana").Strategy;
+
+const express = require("express");
+const session = require("express-session");
+const app = express();
+const port = 3000;
+
+let client = Asana.ApiClient.instance;
+let token = client.authentications["token"];
+// token.accessToken = process.env.PAT;
+
+// https://github.com/mjhea0/passport-social-auth/blob/master/server/app.js
+
+passport.serializeUser(function (user, done) {
+  done(null, user);
+});
+
+passport.deserializeUser(function (obj, done) {
+  done(null, obj);
+});
+
+passport.use(
+  new AsanaStrategy(
+    {
+      clientID: process.env.CLIENT_ID,
+      clientSecret: process.env.CLIENT_SECRET,
+      callbackURL: "https://t8dmgz-3000.csb.app/auth/asana/callback",
```

Comment:
If this is code that will be shown to external devs to reference, might want to set the values to generic templates 

```
      clientID: <CLIENT ID>,
      clientSecret: <CLIENT SECRET>,
      callbackURL: "https://your-app-domain/auth/asana/callback",
```